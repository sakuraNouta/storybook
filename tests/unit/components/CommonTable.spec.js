import { shallowMount, createLocalVue } from '@vue/test-utils';
import CommonTable from '@/views/common-table';
import ElementUI from 'element-ui';

const localVue = createLocalVue();
localVue.use(ElementUI);

describe('CommonTable', () => {
  it('Render', () => {
    const wrapper = shallowMount(CommonTable, {
      localVue
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
