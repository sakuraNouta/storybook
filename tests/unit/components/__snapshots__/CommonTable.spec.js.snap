// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonTable Render 1`] = `
<div class="common-table">
  <div class="common-table__header">
    <div class="common-table__search">
      <el-input size="small" placeholder="搜索租户名称"><i slot="suffix" class="el-input__icon el-icon-search"></i></el-input>
    </div>
  </div>
  <div class="common-table__content">
    <el-table data="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
      <el-table-column label="租户名" prop="name"></el-table-column>
      <el-table-column label="邮箱" prop="email"></el-table-column>
      <el-table-column label="联系方式" prop="telephone"></el-table-column>
    </el-table>
  </div>
  <div class="common-table__footer">
    <el-pagination small="" layout="prev, pager, next" total="100" page-size="10" current-page="1"></el-pagination>
  </div>
</div>
`;
